### PR TITLE
fix: fixing schemas for Servicetoken and accesspoint.interface.

### PR DESCRIPTION
**Pull request by srushti-patl**
This PR includes fixes for two defects(CXF-[81949](https://equinixjira.atlassian.net/browse/CXF-81949) & [CXF-81844)](https://equinixjira.atlassian.net/browse/CXF-81844)
- fixed service token schema
- fixed accespoint.interface schema

### CODE REVIEW

File: equinix/resource_fabric_connection.go [hunk 1]
```diff
@@ -50,7 +50,7 @@ func resourceFabricConnectionCreate(ctx context.Context, d *schema.ResourceData,
 	projectReq := d.Get("project").(*schema.Set).List()
 	project := projectToFabric(projectReq)
 	additionalInfo := d.Get("additional_info").([]interface{})
-	additionalinfo := additionalInfoToFabric(additionalInfo)
+	additionalinfoGoToTerraState := additionalInfoToFabric(additionalInfo)
```
Comment (thogarty) [inline equinix/resource_fabric_connection.go:53]: This is actually `additionalInfoTerraConfigToGo`. The i should be capitalized in info and the data flow is Terraform Config -> Go Struct.

However, the variable shouldn't be the one naming the operation, the method should be. Everywhere we have `ToFabric` should become `TerraToGo` in the methods themselves. 

I would name the variable on line 52 `additionalInfoTerraConfig` and make the variable on line 53 be `additionalInfo`.

If you want to change the method name with this PR you can but not necessary.

**Review by thogarty — APPROVED** [review]
Just the one requested change. Looks good aside from that.

File: equinix/resource_fabric_connection.go (hunk 1 above)
Comment (srushti-patl) [reply equinix/resource_fabric_connection.go]: I have changed both the variable names and the method name.

**Review by thogarty — APPROVED** [review]
LGTM. Ship it.

**Pull request merged**